includes/class-wc-stripe-helper.php

```diff
@@ -315,6 +315,39 @@ public static function get_minimum_amount() {
 		return $minimum_amount;
 	}
 
+	/**
+	 * Gets the supported card brands, taking the store's base country and currency into account.
+	 * For more information, please see: https://stripe.com/docs/payments/cards/supported-card-brands.
+	 *
+	 * @return array
+	 */
+	public static function get_supported_card_brands() {
+		$base_country = wc_get_base_location()['country'];
+		$base_currency = get_woocommerce_currency();
+
+		$supported_card_brands = [ 'visa', 'mastercard' ];
+
+		// American Express is not supported in Brazil and Malaysia.
+		if ( ! in_array( $base_country, [ 'BR', 'MY' ] ) ) {
```

Comment:
We still need to provide support for PHP 5.2 in Stripe, so we need to use the old school array syntax:

```php
array( 'BR', 'MY' )
```
instead of

```php
[ 'BR', 'MY' ]
```

Can you update it?

Reply:
Thanks! I wasn't aware of the need to support PHP 5.2 so that's good to know.